src/rdkafka_int.h

```diff
@@ -29,11 +29,14 @@
 #ifndef _RDKAFKA_INT_H_
 #define _RDKAFKA_INT_H_
 
-#ifndef _MSC_VER
+#ifndef _WIN32
 #define _GNU_SOURCE  /* for strndup() */
-#else
+#endif
+
+#ifdef _MSC_VER
```

Comment:
I'm guessing this is intentional because mingw provides mode_t and msvc does not?

Reply:
Yes.